# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ChessbotActionGoal action_goal
ChessbotActionResult action_result
ChessbotActionFeedback action_feedback
